llvm/lib/Target/SPIRV/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ add_llvm_target(SPIRVCodeGen
   SPIRVRegularizer.cpp
   SPIRVSubtarget.cpp
   SPIRVTargetMachine.cpp
+  SPIRVTargetTransformInfo.cpp
```

Comment:
Hi @jzc 

Can you please try a shared libs build and verfy that there are no build errors?
BUILD_SHARED_LIBS=ON

Thanks